17style-elements/README.md

```diff
@@ -0,0 +1,14 @@
+![phillip zastrow logo](https://pbs.twimg.com/profile_images/1452633114044403715/d3liT5vd_400x400.jpg)
+# [*Tutorial 17*](https://www.digitalocean.com/community/tutorials/how-to-style-html-elements-with-borders-shadows-and-outlines-in-css) 
+## How To Style HTML Elements with Borders, Shadows, and Outlines in CSS
+
+## What I learned and how it applies in my Github page example:
+- The `border-property` applies a line in any color to the outer perimeter of a container. The properties for border are: thickness, style, and color. 
+- You can specify individual sides that will have a border with the direction variations like `border-right`, etc.
+- Border styles can create depth and define a container. Adding `border-color` works well for this.
+- The way to define width and style is with `border`. Example: `border: 1px solid`.
+- Borders can be further specified with combinations of these properties such as `border-right-color`.
+- With `text-shadow` you can specify shadows on text content.
+- The `box-shadow` property allows you to add shadows to elements and containers.
+- You can apply an outline to a button with `.button: focus`. By default browsers outline buttons with a certain look so that when you tab to them they appear to be focused on with an outline.
+- If you set the button `outline:none` on the button, it allows you to set more customized styles with the `.button: focus {}` selector.
```

Comment:
There shouldn't be a space between `.button:` and `focus`. For hover and focus states (among others), the colon prefix needs to be applied like `:hover` and `:focus`. Same with pseudo-selectors like `::before` and `::after`.

You can use [this tool](https://kittygiraudel.github.io/selectors-explained) to show how your selectors are understood in plain language. If you try `.button: focus`, you can see that it breaks, but `.button:focus` works, and `.button :focus` gives you something completely different.

Reply:
That is a great link, I really appreciate it! I will go in and fix this. 👍